### PR TITLE
Add badge section to docs landing

### PR DESCRIPTION
**Pull request by spalladino**
Adds a new section prompting the user to add a badge to their READMEs. I'm open to discussing whether this is too much self-promotion or not!

### CODE REVIEW

**Comment by spalladino** [comment]
Hah, PR #100!

File: components/home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -84,3 +84,14 @@ You can https://forum.openzeppelin.com/[join our Forum], https://blog.openzeppel
 == Questions?
 
 Join the OpenZeppelin community in https://forum.openzeppelin.com/[our forum] to https://forum.openzeppelin.com/c/support/17[ask for help], as well as share your feedback or showcase what you have built with OpenZeppelin!
+
+== Share the love!
```
Comment (nventuro) [inline components/home/modules/ROOT/pages/index.adoc]:
```suggestion
== Share the Love!
```

**Comment by nventuro** [comment]
I love this! I feel the color should be a bit lighter, but it rocks!

**Comment by spalladino** [comment]
The color was set by our dear @blancoagostina, I wouldn't dare change it.

**Pull request merged**